src/gui/settingsDialog/TrayIconSettings.cpp

```diff
@@ -74,6 +77,7 @@ void TrayIconSettings::initGui()
 	mStartMinimizedToTrayCheckBox->setText(tr("Start Minimized to Tray"));
 	mCloseToTrayCheckBox->setText(tr("Close to Tray"));
 	mTrayIconNotificationsCheckBox->setText(tr("Display Tray icon notifications"));
+	mUsePlatformSpecificNotificationServiceCheckBox->setText(tr("Use platform specific notification service"));
```

Comment:
Can you add a tooltip that says that changes of this feature require a ksnip restart?